src/makeCalculator.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+      return this;
+    },
+
+    subtract(value) {
+      this.result -= value;
+      return this;
+    },
+
+    multiply(value) {
+      this.result *= value;
+      return this;
+    },
+
+    divide(value) {
+      this.result /= value;
```

Comment:
There is a potential issue in the `divide` method. Dividing by zero will result in `Infinity` or `NaN`, which might not be the desired behavior. Consider adding a check to handle division by zero appropriately.